src/main/resources/definitions/prisons/test/test-report.json

```diff
@@ -18,7 +18,7 @@
       "id": "test",
       "name": "Test data",
       "datasource": "redshift",
-      "query": "SELECT 'One' as column1, 2 as column2, 'Three' as column3, TO_TIMESTAMP('2004-05-06T07:08:09Z', 'YYYY-MM-DDTHH24:MI:ssZ') as column4 UNION SELECT 'Remove' as column1, null as column2, null as column3, null as column4 UNION SELECT 'Eleven' as column1, 12 as column2, 'Thirteen' as column3, TO_TIMESTAMP('2014-12-16T17:18:19Z', 'YYYY-MM-DDTHH24:MI:ssZ') as column4",
+      "query": "SELECT 'One' as column1, 2 as column2, 'Three' as column3, TO_TIMESTAMP('2004-05-06 07:08:09', 'YYYY-MM-DD HH24:MI:ss') as column4 UNION SELECT 'Remove' as column1, null as column2, null as column3, null as column4 UNION SELECT 'Eleven' as column1, 12 as column2, 'Thirteen' as column3, TO_TIMESTAMP('2014-12-16 17:18:19', 'YYYY-MM-DD HH24:MI:ss') as column4",
       "schema": {
```

Comment:
These changes look odd. This is the test report used for acceptance tests. Are these changes to support a format change or such?